HLTrigger/special/src/HLTEcalPhiSymFilter.cc

```diff
@@ -43,11 +49,19 @@ HLTEcalPhiSymFilter::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   desc.add<edm::InputTag>("endcapHitCollection",edm::InputTag("ecalRecHit","EcalRecHitsEE"));
   desc.add<unsigned int>("statusThreshold",3);
   desc.add<bool>("useRecoFlag",false);
-  desc.add<double>("ampCut_barrel",8.);
-  desc.add<double>("ampCut_endcap",12.);
+  desc.add<std::vector<double> >("ampCut_barrelP",{6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
+                                                   6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
+                                                   6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.});
+  desc.add<std::vector<double> >("ampCut_barrelM",{6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
+                                                   6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,
+                                                   6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.,6.});
+  desc.add<std::vector<double> >("ampCut_endcapP",{9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,
+                                                   9.,9.,9.,9.,9.});
+  desc.add<std::vector<double> >("ampCut_endcapM",{9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,9.,
+                                                   9.,9.,9.,9.,9.});
   desc.add<std::string>("phiSymBarrelDigiCollection","phiSymEcalDigisEB");
   desc.add<std::string>("phiSymEndcapDigiCollection","phiSymEcalDigisEE");
-  descriptions.add("alCaPhiSymStream",desc);
+  descriptions.add("AlCaPhiSymStream",desc);
```

Comment:
It has always been that name, I've only changed the capital letter. If needed I can change it to be hltEcalPhiSymFilter.